src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js

```diff
@@ -347,11 +347,12 @@ class DatagridCell extends React.Component {
     const { aria } = this.props;
     const style = this.getStyle();
     const modifiers = this.getModifiers();
+    const value = this.getFormattedValue();
 
     return (
       <div
         className={bemClass('DatagridCell', modifiers, this.props.className)}
-        title={this.getFormattedValue()}
+        title={[undefined, null].includes(value) ? '' : value.toString()}
```

Comment:
Yesterday I discuses about similarly solution with Serhii.
I don't like code like this. We are not able predict what we should display and what not.
What if there will be `NaN` ? 
I think we should create some helper `getTitleValue`:
```
(value) => {
  if (!value && !== 0) {
    return '';
  }

  return value.toString();
}
```

Then if we can use it in other places and we have full control in one place when need some other conditions

Reply:
I'm agree with helper, but don't understand code above. In that case we are displaying true as "true" but false as "" (on hover checkbox item). 
@aleksandranovakovic can you specify what we should display on hover cell in grid when value is: 
- empty (null/undefined) 
- NaN
- false/true (checkboxes)

Reply:
I moved it to helper, but this solution is not correct